src/python/qepsi4/_psi4.py

```diff
@@ -2,44 +2,45 @@
 import numpy as np
 from openfermion import InteractionOperator, general_basis_change, MolecularData
 from openfermion.config import EQ_TOLERANCE
+from typing import Tuple
 
 
 def run_psi4(
-    geometry,
-    basis="STO-3G",
-    multiplicity=1,
-    charge=0,
-    method="scf",
-    reference="rhf",
-    freeze_core=False,
-    n_active=None,
-    save_hamiltonian=False,
-    options=None,
-    n_active_extract=None,
-    n_occupied_extract=None,
-    freeze_core_extract=False,
-):
+    geometry: dict,
+    basis: str = "STO-3G",
+    multiplicity: int = 1,
+    charge: int = 0,
+    method: str = "scf",
+    reference: str = "rhf",
+    freeze_core: bool = False,
+    n_active: int = None,
+    save_hamiltonian: bool = False,
+    options: dict = None,
+    n_active_extract: int = None,
+    n_occupied_extract: int = None,
+    freeze_core_extract: bool = False,
+) -> Tuple[dict, InteractionOperator]:
```

Comment:
Would it be better to specify a variable-length Tuple in case we want to return some other objects in addition to the results dictionary and hamiltonian? (e.g. RDMs, irreps list, additional information about  the active space, etc)